eslint-plugin/README.md

```diff
@@ -0,0 +1,38 @@
+# eslint-plugin-tsdoc
+
+This ESLint plugin provides a rule for validating that TypeScript doc comments conform to the
+[TSDoc specification](https://github.com/microsoft/tsdoc).
+
+## Usage
+
+1. Configure ESLint for your TypeScript project.  See the instructions provided by the
+   [typescript-eslint](
+   https://github.com/typescript-eslint/typescript-eslint#how-do-i-configure-my-project-to-use-typescript-eslint) project.
+
+2. Add the `eslint-plugin-tsdoc` dependency to your project:
+
+    ```bash
+    $ cd my-project
+    $ npm install --save-dev eslint-plugin-tsdoc
+    ```
+
+3. Enable the rule in your ESLint config file.  Example usage:
+
+    **my-project/.eslintrc.js**
```

Comment:
Missing configuration for the plug-in?

Reply:
I've updated the README.md based on testing